geo-types/src/geometry.rs

```diff
@@ -182,94 +181,56 @@ impl<T: CoordNum> Geometry<T> {
     }
 }
 
-#[derive(Debug)]
-pub struct FailedToConvertError;
```

Comment:
I deleted this FailedToConvertError.

My understanding is that this is *not* a breaking change, because, though it was public, it was never exported - so noone could have typed in "geo_types::FailedToConvertError" or be matching on that explicit case... so non-breaking, right?